NPM install doesn't work [BUG]

# Context

* Package version: last
* Node version: 22

# Steps to Reproduce

1. checkout to node 22
2. npm install @prestashopcorp/puik-theme @prestashopcorp/puik-components @prestashopcorp/puik-resolver --save

# Expected Behavior

Success installation

# Current Behavior

Errors: 

> npm install @prestashopcorp/puik-theme @prestashopcorp/puik-components @prestashopcorp/puik-resolver --save
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*

npm ERR! A complete log of this run can be found in: /home/me/.npm/_logs/2026-04-10T08_30_36_086Z-debug-0.log

--- a/.github/workflows/cd-publish-release.yml
+++ b/.github/workflows/cd-publish-release.yml
@@ -58,7 +58,9 @@ jobs:
                 fi
                 echo "Publishing $NAME@$VERSION"
                 cd "$pkg"
-                npm publish --access public --provenance
+                TARBALL=$(pnpm pack 2>/dev/null | tail -1)
+                npm publish "$TARBALL" --access public --provenance
+                rm -f "$TARBALL"
                 cd "$GITHUB_WORKSPACE"
               fi
             fi

